src/micromamba/main.cpp

```diff
@@ -915,6 +927,12 @@ init_install_global_parser(CLI::App* subcom)
         ->allow_extra_args(false);
     subcom->add_flag(
         "--no-pin", create_options.no_pin, "Ignore pinned packages (from config or prefix file)");
+    subcom->add_flag(
+        "--allow_softlinks", create_options.allow_softlinks, "Allow softlinks when hardlinks fail");
```

Comment:
I prefer `--allow-softlinks` vs underscore

Reply:
it's a typo sorry, will be fixed!